.github/workflows/checks.yaml

```diff
@@ -96,10 +108,21 @@ jobs:
         env:
           RUST_BACKTRACE: 1
       - name: cargo nextest
-        if: matrix.os == 'windows-2022'
+        if: matrix.target == 'x86_64-pc-windows-gnullvm'
+        run: |
+          echo "llvm-mingw/llvm-mingw-20250709-ucrt-x86_64/bin" >> "$GITHUB_PATH"
+          echo "llvm-mingw/llvm-mingw-20250709-ucrt-x86_64/lib" >> "$GITHUB_PATH"
+          echo "llvm-mingw/llvm-mingw-20250709-ucrt-x86_64/armv7-w64-mingw32/bin" >> "$GITHUB_PATH"
```

Comment:
This should be the x86_64 path, not the armv7 library path, right

Reply:
eh yes. but also I don't think writing to the github_path would actually have much effect here. 

clearly, I have no idea what I'm doing here, and it's tough to debug in CI